src/main/scala/com/vngrs/etl/Pipeline.scala

```diff
@@ -0,0 +1,94 @@
+package com.vngrs.etl
+
+import com.vngrs.etl.combiners.{Union, Zip}
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+/**
+  * Represents ETL Pipeline.
+  *
+  * @param rdd The [[org.apache.spark.rdd.RDD]] which will be ETLed
+  * @tparam A Type of the [[org.apache.spark.rdd.RDD]]
+  */
+final class Pipeline[A: ClassTag](val rdd: RDD[A]) {
```

Comment:
Actually, it does not. With this current design, `Extract`, `Transform` and `Load` is totally abstracted from `Pipeline`. So anyone who wishes to use this library can just use implemented extractors not the whole pipeline. Also, since there is no implemented extractors yet it seems like a lonely trait but it will not in the future.